### PR TITLE
Fix/get rid of loading text

### PR DESCRIPTION
**Pull request by gwkim8107**
___
NB - by making a pull request you confirm agreement to the [Contributors Agreement](https://github.com/openlawnz/openlawnz-web/blob/master/CONTRIBUTING.md).

I did remove the loading text which was shown on every page, and this issue is regarding an issue of Get rid of loading text.

### CODE REVIEW

File: src/jsx/App.jsx [hunk 1]
```diff
@@ -28,7 +28,8 @@ const App = props => {
 	const updateNewsData = news => setNews(news);
 
 	if (loading) {
-		return <div>Loading...</div>;
+		// In relevant with an issuie of Get rid of loading text, below <div> should be commented or removed
+		// return <div>Loading...</div>;
 	}
```
Comment (caionnx) [inline src/jsx/App.jsx]: @gwkim8107 you can remove the whole `if` and comment the lines 26 and 18. We'll not need the `const` nor the `import` right now.

File: README.md [hunk 2]
```diff
@@ -18,14 +17,9 @@ Visit http://localhost:9000/
...
-https://github.com/openlawnz/openlawnz-web/wiki/OpenLawNZ-Website-Overview
-
-Contributing agreement and code of conduct:
-https://github.com/openlawnz/openlawnz-web/blob/master/CONTRIBUTING.md
-
-Workflow:
-https://github.com/openlawnz/openlawnz-web/wiki/Workflow
+* [Overview](https://github.com/openlawnz/openlawnz-web/wiki/OpenLawNZ-Website-Overview)
+* [Contributing agreement and code of conduct](https://github.com/openlawnz/openlawnz-web/blob/master/CONTRIBUTING.md)
+* [Workflow](https://github.com/openlawnz/openlawnz-web/wiki/Workflow)
```
Comment (caionnx) [inline README.md]: @gwkim8107 are the changes in this file related to the issue?

Comment (caionnx) [reply README.md]: Oh, I saw that Will made those changes, I think you just need to `pull origin development` and push it again.

Comment (gwkim8107) [reply README.md]: I've completed for deleting those relating with 'loading' variable and git push to the branch 'fix/get-rid-of-loading-text'.

File: src/jsx/App.jsx (hunk 1 above)
Comment (gwkim8107) [reply src/jsx/App.jsx]: Deleted lines between 18 and 26.

**Pull request merged**